resources/text/menu.json

```diff
@@ -0,0 +1,44 @@
+{
+  "starting_menu": [
+    "1. Travel the Trail",
```

Comment:
The number should not be hard coded in the json file so we can change the menu more easily if we want to.

```suggestion
    "%d. Travel the Trail",
```

Reply:
I need a little bit more feedback on this one.  I don't understand why we can't have the number hard coded in the JSON.  If I have the number inserted through string interpolation I have to do something like this in the code, which does not look easy to maintain. 
![image](https://user-images.githubusercontent.com/23693901/181357449-ea6cb370-0e6a-4c4a-a139-1e71f232eb32.png)
 If we reorder anything from the menu it the lines would have to be moved around in the JSON anyway otherwise it looks weird.

Reply:
This is a minor one. In general its best practice to avoid redundancy (numbering a ordered list) and this is a bigger issue when the list is long and you expect that you would reorder or add items to the list often, which isn't the case. You can change it back if you would like